website/docs/plugins/optuna_sweeper.md

```diff
@@ -208,3 +208,46 @@ If `log` is `false`, the parameter is mapped to [`UniformDistribution`](https://
   - `choices`: a list of parameter value candidates
 
 The parameters are mapped to [`CategoricalDistribution`](https://optuna.readthedocs.io/en/stable/reference/generated/optuna.distributions.CategoricalDistribution.html).
+
+
+## Example 2:  Multi-Objective Optimization
+
+We include another example in the directory. `example/multi-objective.py` implements a simple benchmark function which has two objective values. We'd like to minimize both objectives simultaneously.
+
+You can discover the Optuna sweeper parameters with:
+
+```yaml title="python example/multi-objective.py hydra/sweeper=optuna --cfg hydra -p hydra.sweeper"
```

Comment:
use <details> tag to fold this output.
See example in other plugins.

Reply:
I applied the `details` tag in the commit https://github.com/facebookresearch/hydra/pull/1294/commits/0f774bc2d1fad317b1a3bb1876df760642372159. Thank you!
![image](https://user-images.githubusercontent.com/3255979/106156244-a4b8f080-61c4-11eb-8449-35bd78109c53.png)